context/src/test/java/io/grpc/ContextTest.java

```diff
@@ -967,7 +967,7 @@ public void close() throws SecurityException {
         assertNull(logRef.get());
         ctx = ctx.fork();
       }
-      ctx = ctx.fork();
+      ctx.fork();
```

Comment:
Why?

Reply:
ctx is unused which ErrorProne complains about

Reply:
I see, we had `@SuppressWarnings("CheckReturnValue") ` for `ContextTest`, so there is no problem.